GitHubUsers.ps1

```diff
@@ -387,6 +380,13 @@ function Set-GitHubProfile
     if ($PSBoundParameters.ContainsKey('Bio')) { $hashBody['bio'] = $Bio }
     if ($PSBoundParameters.ContainsKey('Hireable')) { $hashBody['hireable'] = $Hireable.ToBool() }
 
+    if (-not $PSCmdlet.ShouldProcess('Update Current GitHub User'))
+    {
+        return
+    }
+
+    Write-InvocationLog
```

Comment:
Double `Write-InvocationLog` (here and at 372)

Reply:
First one removed.